scripts/find_spammy_content.py

```diff
@@ -45,7 +47,7 @@ def manage_spammy_content(regex, days=1, models=[Node, Preprint, Registration],
                 item['username'],
                 item['fullname']
             ])
-        return writer
+        return output
```

Comment:
If ever executed from command line, you want the `IOStream` for `outFile.write(output.getValue()`, not the `csvWriter`. Return value discarded in admin.